sdks/python/apache_beam/yaml/standard_providers.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+- type: 'beamJar'
+  gradle_target: 'sdks:java:extensions:sql:expansion-service:shadowJar'
+  version: '2.43.0'
+  transforms:
+     Sql: 'beam:external:java:sql:v1'
```

Comment:
What else will be in the standard providers ? All Schem-Aware transforms provided by Beam ?

If latter, it might be good to dynamically generate this (via ExpansionService.discover()) instead of maintaining a static list.

Reply:
Yep, I would be in favor of automatically generating this.